stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -540,17 +540,28 @@ extension Tensor where Scalar : TensorFlowFloatingPoint {
   ) -> (Tensor, (Tensor) -> Tensor) {
     let value = reshaped(toShape: newShape)
     return (value, { v in
-      return v.reshaped(toShape: self.shapeTensor)
+      return v.scalarCount == 1 ?
+        v.broadcast(toShape: self.shapeTensor) :
+        v.reshaped(toShape: self.shapeTensor)
     })
   }
 
+  @inlinable
+  func _vjpSqueezingShape(at axes: Int32...) -> (Tensor, (Tensor) -> Tensor) {
+    // Cannot call squeezingShape(at:) since it has variadic arguments
```

Comment:
Could you please create a non-variadic version `func squeezingShape(at axes: [Int32])` and define `_vjpSqueezingShape(at axes: [Int32])` for it?